tests/testthat/test-Data-methods.R

```diff
@@ -23,3 +23,19 @@ test_that("Plot works as expected for Data object with placebo and with blinding
 
   vdiffr::expect_doppelganger("Plot of Data with placebo and with blinding", result)
 })
+
+# plot-DataDual ----
+
```

Comment:
maybe would be good to add one more test for the `Data` plot method now that you can have the legend switched off?